app/utils/HouseConsignmentAnswersHelper.scala

```diff
@@ -18,31 +18,29 @@ package utils
 
 import cats.data.OptionT
 import cats.implicits._
+import models.departureTransportMeans.TransportMeansIdentification
+import models.reference.Country
 import models.{Index, UserAnswers}
 import pages._
 import pages.houseConsignment.index.items.ItemDescriptionPage
 import pages.sections._
 import pages.sections.departureTransportMeans.DepartureTransportMeansListSection
 import play.api.i18n.Messages
-import services.ReferenceDataService
 import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
 import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
-import uk.gov.hmrc.http.HeaderCarrier
 import viewModels.sections.Section
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class HouseConsignmentAnswersHelper(userAnswers: UserAnswers, houseConsignmentIndex: Index, referenceDataService: ReferenceDataService)(implicit
+class HouseConsignmentAnswersHelper(userAnswers: UserAnswers, houseConsignmentIndex: Index)(implicit
   messages: Messages,
-  hc: HeaderCarrier,
   ec: ExecutionContext
 ) extends UnloadingAnswersHelper(userAnswers) {
 
   def buildVehicleNationalityRow(transportMeansIndex: Index): Future[Option[SummaryListRow]] =
     (for {
       x <- OptionT.fromOption[Future](userAnswers.get(DepartureTransportMeansCountryPage(houseConsignmentIndex, transportMeansIndex)))
-      y <- OptionT.liftF(referenceDataService.getCountryNameByCode(x))
-    } yield transportRegisteredCountry(y)).value
+    } yield transportRegisteredCountry(x)).value
```

Comment:
I suspect this can be refactored? We don't need this method to return a Future anymore